src/bp-core/bp-core-options.php

```diff
@@ -2263,3 +2274,59 @@ function bb_get_default_custom_upload_group_cover() {
 	 */
 	return apply_filters( 'bb_get_default_custom_upload_group_cover', bp_get_option( 'bp-default-custom-group-cover' ) );
 }
+
+/**
+ * Get profile cover image width.
+ *
+ * @since BuddyBoss [BBVERSION]
+ *
+ * @param string|null $default Optional. Fallback value if not found in the database.
+ *                             Default: 'default'.
+ *
+ * @return string Return profile cover image width.
+ */
+function bb_get_profile_cover_image_width( $default = 'default' ) {
+	return bp_get_option( 'bp-cover-profile-width', $default );
+}
```

Comment:
@milanhirapra This should go to the members function file.

Reply:
done